script/setup

```diff
@@ -1,53 +0,0 @@
-#!/usr/bin/env sh
-
-# Set up Rails app. Run this script immediately after cloning the codebase.
-#
-# First, you need to create the database user the app will use by manually
-# typing the following in your terminal:
-#
-#   $ sudo -u postgres psql -c "CREATE USER ofn WITH SUPERUSER CREATEDB PASSWORD 'f00d'"
-#
-# This will create the "ofn" user as superuser and allowing it to create
-# databases.
-
-# Exit if any command fails
-set -e
-
-YELLOW='\033[0;33m'
-RED='\033[0;31m'
-NO_COLOR='\033[0m'
-
-RUBY_VERSION=$(cat .ruby-version)
-
-# Check ruby version
-if ! ruby --version | grep $RUBY_VERSION > /dev/null; then
```

Comment:
This check seems pretty useful. I would suggest we keep it.

I can also see possibility to improve and add a node version check also.
Maybe even conditionally run rbenv and nodenv if installed.

Reply:
Good point. When I removed that I thought that we can just run `script/rbenv-install` but then I realised that this `bin/setup` is a Ruby script and won't run before installing ruby. And even when Ruby and rbenv are present then rbenv complaints that it can't find the right version.

So maybe that's another reason to keep script/setup to run the ruby installation.